option cc_generic_services = true;

message AddRequest {
	required int32 input = 1;
}

message AddResponse {
	required int32 result = 1;
}

service AddService {
	rpc add(AddRequest) returns(AddResponse);
}
